ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/AgentUtils.java

```diff
@@ -149,7 +149,7 @@ public static String extractModelResponseJson(String text) {
         if (matcher.find()) {
             return matcher.group(1);
         } else {
-            throw new IllegalArgumentException("Model output is invalid");
+            throw new IllegalArgumentException("Model output is invalid:" + text);
```

Comment:
What do you think about: `throw new IllegalArgumentException("Invalid model output: '" + text + "' does not meet the required criteria.");`